src/common/types/gristData.ts

```diff
@@ -7,7 +17,7 @@ export interface TableRowType {
     Einrichtung: string
     Trager: string
     Kategorie: string
-    Schlagworte: string
+    Schlagworte: number[]
```

Comment:
So, when fetching the choice list field programmatically, we need to work with numbers i.e. ID's? How can we map the ID to a label then? Or do we not need that? Maybe I'm not getting the context right here.

Reply:
The reason is that I use another table for the labels now which returns an array of number IDs. We then use the id of the active filters to map through the records and check if the label ID is present in the record. See the following code in `FilterList.tsx`:

```ts
const filteredRecords = recordsWithOnlyLabels.filter((r) =>
    activeFilters.every((f) => r.find((id) => id === f.id))
  )
```

Reply:
Alright, thanks for clarifying 🙂